client/state/posts/revisions/selectors.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import { get, map } from 'lodash';
+import createSelector from 'lib/create-selector';
+
+/**
+ * Internal dependencies
+ */
+import { normalizePostForDisplay } from '../utils';
+
+export const getPostRevisions = createSelector(
+	( state, siteId, postId ) => {
+		return map(
+			get( state.posts.revisions.revisions, [ siteId, postId ], [] ),
+			normalizePostForDisplay );
```

Comment:
Minor: I guess it's personal preference, but I like to keep closing/opening parens on their own line
 when we have long params like here.

```js 
return map(
  get( state.posts.revisions.revisions, [ siteId, postId ], [] ),
  normalizePostForDisplay
);
```

Same above with the if

```js
if (
  this.props.siteId === nextProps.siteId &&
  this.props.postId === nextProps.postId 
) {
```